modules/rest-api/pages/rest-get-memory-information.adoc

```diff
@@ -0,0 +1,100 @@
+= Getting Memory Information
+:description: pass:q[Use REST API to retrieve the current memory quota information for services in a cluster.]
+:page-toclevels: 2
+
+[abstract]
+{description}
+
+[#http-method-and-uri]
+== HTTP Method and URI
+
+----
+GET /pools/default
+----
+
+[#description]
+== Description
+
+Allows to view memory quotas for the Multi-Dimensional Scaling (MDS) services Index, Search, Eventing, and Analytics.
+Query and Backup services do not have a memory allocation.
+If no custom quota was specified during configuration for one or more services, those services retain the default allocations.
+These are the default allocations:
+
+* 256 Mb for fts (Search Service)
```

Comment:
Need Data Service.

* 256 Mb for kv (Data Service)
* 256 Mb for fts (Search Service)